crates/forge/src/scarb.rs

```diff
@@ -29,9 +31,42 @@ pub fn config_from_scarb_for_package(
     raw_config.try_into()
 }
 
+pub fn build_contracts_with_scarb(filter: PackagesFilter) -> Result<()> {
+    let build_output = Command::new("scarb")
+        .arg("build")
+        .env("SCARB_PACKAGES_FILTER", filter.to_env())
+        .stderr(Stdio::inherit())
+        .stdout(Stdio::inherit())
+        .output()
+        .context("Failed to build contracts with Scarb")?;
+
+    if build_output.status.success() {
+        Ok(())
+    } else {
+        Err(anyhow!("Scarb build did not succeed"))
+    }
+}
+
+pub fn build_test_artifacts_with_scarb(filter: PackagesFilter) -> Result<()> {
+    let build_output = Command::new("scarb")
+        .arg("snforge-test-collector")
+        .env("SCARB_PACKAGES_FILTER", filter.to_env())
+        .stderr(Stdio::inherit())
+        .stdout(Stdio::inherit())
+        .output()
+        .context("Failed to build test artifacts with Scarb")?;
+
+    if build_output.status.success() {
+        Ok(())
+    } else {
+        Err(anyhow!("scarb snforge-test-collector did not succeed"))
```

Comment:
```suggestion
        Err(anyhow!("Scarb snforge-test-collector did not succeed"))
```

Reply:
It is a binary call i.e. `scarb snforge-test-collector`

Reply:
We capitalize scarb in the other function, so I'd at least change them to be consistent with each other